api_tests/metaschemas/views/test_metaschemas_list.py

```diff
@@ -1,38 +1,39 @@
-from nose.tools import *  # flake8: noqa
-
-from osf.models.metaschema import MetaSchema
-from website.project.metadata.schemas import LATEST_SCHEMA_VERSION
+import pytest
 
 from api.base.settings.defaults import API_BASE
-
-from tests.base import ApiTestCase
+from osf.models.metaschema import MetaSchema
 from osf_tests.factories import (
     AuthUserFactory
 )
+from website.project.metadata.schemas import LATEST_SCHEMA_VERSION
+
+@pytest.mark.django_db
+class TestMetaSchemaList:
```

Comment:
Again, since this class only has one test now I'm not sure if fixtures are needed